Juice.xcodeproj/project.pbxproj

```diff
@@ -490,17 +500,38 @@
 			isa = PBXVariantGroup;
 			children = (
 				72DA73611E09FE19003F9262 /* Base */,
+				AA66D5481E0C698700D83161 /* fr */,
 			);
 			name = MainMenu.xib;
 			sourceTree = "<group>";
 		};
+		AA66D5461E0C697E00D83161 /* Localizable.strings */ = {
+			isa = PBXVariantGroup;
+			children = (
+				AA66D5451E0C697E00D83161 /* en */,
+				AA66D5491E0C698700D83161 /* fr */,
+			);
+			name = Localizable.strings;
+			sourceTree = "<group>";
+		};
+		AA66D5581E0C77EE00D83161 /* GeneralPreferencesViewController.xib */ = {
+			isa = PBXVariantGroup;
+			children = (
+				AA66D5571E0C77EE00D83161 /* en */,
+				AA66D55B1E0C77F700D83161 /* fr */,
+			);
+			name = GeneralPreferencesViewController.xib;
+			path = ../../.Trash/Juice/Classes;
```

Comment:
This path seems wrong, should this be pointing to the file located within the project?